golang/go-cloud-run-hello-world/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Use base golang image from Docker Hub
+FROM golang:1.13
+
+# Download the dlv (delve) debugger for go (you can comment this out if unused)
+RUN go get -u -v github.com/go-delve/delve/cmd/dlv
+
+WORKDIR /src/hello-world
+
+# Install dependencies in go.mod and go.sum
+COPY go.mod go.sum ./
+RUN go mod download
+
+# Copy rest of the application source code
+COPY . ./
+
+# Compile the application to /app.
+RUN go build -mod=readonly -v -o /app
+
+# Copy template & assets
+COPY index.html /index.html
+COPY assets ./assets/
+
+# If you want to use the debugger, you need to modify the entrypoint to the
+# container and point it to the "dlv debug" command:
+#   * UNCOMMENT the following ENTRYPOINT statement,
+#   * COMMENT OUT the last ENTRYPOINT statement
+# Start the "dlv debug" server on port 3000 of the container.
+ENTRYPOINT ["dlv", "exec", "/app", "--continue", "--accept-multiclient", "--api-version=2", "--headless", "--listen=:3000", "--log"]
+
+# If you want to run WITHOUT the debugging server:
+#   * COMMENT OUT the previous ENTRYPOINT statements,
+#   * UNCOMMENT the following ENTRYPOINT statement.
+# ENTRYPOINT ["/app"]
```

Comment:
This file was copied from `go-hello-world`.

Changes include:
* adding static assets
* swapping in the -mod=readonly directive used in the Cloud Run samples.

Normally for golang I'd use a multi-stage Dockerfile, but not sure what the right way is to split the difference with local debugging, so I'm following the existing sample. If we're using this file as a model for a production Dockerfile, it might be worth doing something more complicated.